README.md
# ICT-Shybani
While legal, economic, technological discourses are driving human intellectual discourses through a transformational phase and hence trickling down the effects to general masses globally, Muslim societies are a no exception. Being a dominant civilization, Western Enlightenment provides a cradle to most of such discourses leaving an impact on the individual as well as the collective or societal realms. Unlike many other societies with civilization backgrounds other than Islam, sections of the Muslim societies have attempted to offer a great deal of resistance to such tidings and at the same time struggling to offer an alternative discourse package which is rooted into their own paradigm of Tawhid.  As a next level, it wants to offer solutions to the issues from the perspective of their own knowledge and heritage which once spur from such a paradigm.  Shaybani Foundation would be adding itself into the line of an intellectual corpse by educating the individuals from a multitude of areas to understand the As-Is status of the Muslims in the present day scenario and how to cope with the challenges which are being faced by Muslims and humanity at large. In addition to academic trainings, SY would be perusing for a genuine intellectual activity in various domains and produce adequate literature in the form of books, journals, research articles and so on.  This whole effort is expected to contribute to the overall intellectual activity which would ultimately bring a confidence among Muslims about their heritage, explore the solutions to the issue being faced today and pitch recommendation when and where deemed necessary.


changelog.txt
Version: 5.1.0
  - Updated Bootstrap to version 5.0.0-beta3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to V3.1

Version: 5.0.1
  - Updated Bootstrap to version 5.0.0-beta2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 5.0.0
  - The template does not require jQuery anymore
  - Removed jQuery and all the jQuery plugins
  - The assets/js/main.js was rewritten completely with vanilla Javascript. No more jQuery code
  - Restructured the dev version for better development experience
  - Updated the PHP Email Form to V3.0 - No jQuery dependency. Added attachment support

Version: 4.0.0
  - Initial release with Bootstrap v5.0 Beta 1

Version: 3.2.0
  - Updated Bootstrap to version 4.5.3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to v2.3
  - Other small fixes and improvements

Version: 3.1.0
  - Updated Bootstrap to version 4.5.2
  - Updated the PHP Email Form to v2.1 
  - Updated all outdated third party vendor libraries to their latest versions
  - Other small fixes and improvements

Version: 3.0.0
  - Updated Bootstrap to version 4.5.0
  - Updated the PHP Email Form library to version 2.0 with reCaptcha support
  - Aded inner-page.html tempalte
  - Replaced the WOW animationon scroll library with AOS
  - Rebuilt the header and the navigation section. It's not using the Superfish jQuery plugin anymore
  - Replaced the morphtext library with typed.js in the hero section
  - Added smooth scroll on page load with hash links in the url
  - Added navigation active state on scroll
  - Updated all outdated third party vendor libraries to their latest versions
  - Other small fixes and improvements
  
Version: 2.1.0
  - Improved the portfolio section. Added filterable nav
  - Added portfolio details page
  
Version: 2.0.0
  - The template was rebuilt from scratch with the latest Bootstrap version (4.4.1)
  - Added SMPTP support for the contact form script (Pro)
  - Added NodeJS NPM Development version (Pro unlimited & Membership members)
  
Version: 1.0.0
  - Initial Release


urdu.php
<!DOCTYPE html>
<html>
<head>
  <link rel="stylesheet" media="screen" href="https://fontlibrary.org//face/nafees-nastaleeq" type="text/css"/>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">
<style>
  table thead { display:block; }
  table tbody { height:300px; overflow-y:scroll; display:block; }
  </style>
  <title>Mawdudid's Century</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

 
  <meta name="twitter:image" content="">

  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,700i|Raleway:300,400,500,700,800" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Imperial - v5.1.0
  * Template URL: https://bootstrapmade.com/imperial-free-onepage-bootstrap-theme/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>
<body style="direction : rtl;  font-family: 'NafeesRegular';font-weight: normal; font-style: normal;  
">  
<!-- Header Menu -->
<?php include 'connect.php';

$id=$_GET['id'];
$content=$_POST['content']; //stores content of the chapter that has been clicked in the table of content
$title=$_POST['title']; 
if (isset($_POST['_id'])) {
  
    $id=$_POST['_id'];
}

//stores $id of the book which is selected

$content=$_POST['content']; //stores content of the chapter that has been clicked in the table of content
$title=$_POST['title'];     //stores title value for each chapter
$sql = "SELECT title,content FROM chapters WHERE book_id=".$id; //following snippet extracts name of chapters from the books
$result = $conn->query($sql);  
$sql1 = "SELECT name,title_image,description FROM books WHERE _id=".$id; //extracting details of book (title Image,name and description etc)
$result1 = $conn->query($sql1);  
$row1 = $result1->fetch_assoc()
?>

<header id="header" class="d-flex align-items-center">
<div class="container d-flex align-items-center justify-content-between">
<a href="index.html" class="logo mr-auto"><img src="assets/img/logo.png" alt=""></a>
<nav id="navbar" class="navbar">
        <ul>
          <li><a class="nav-link scrollto active" href="index.php" style="font-family : urdu;">مودودی کی صدی</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->
</div>
</header>
<section id="about">
      <div class="container wow fadeInUp">
        <div class="row">
          <div class="col-md-12">
            
          </div>
        </div>
      </div>
        <div class="row">
            <div class="col-md-auto">
            <?php echo "<img src='./assets/img/portfolio/".$row1["title_image"].".jpg' width='200' hieght='210'>";?>
          </div>
<div class="col-md-6">
            <h2 class="about-title">
        
            <h3 class="section-title"><?php echo $row1["name"];?></h3></h2>
            <div class="section-title-divider"></div>
            <p class="section-description"><?php echo $row1["description"];?></p>
            </div>
        

          <div class="col-md-4" >
            <h2 class="about-title">
            
                <table style="direction:rtl;">
                <thead></thead>
                <tbody>
              <?php if ($result->num_rows > 0) 
{
    while($row = $result->fetch_assoc()) 
{
 //id of the book, and content each chapter storted in a hidden input of a form that is submitted to the same page when clicked on any chapter
 echo "<tr>
            <td>
                <form action='urdu.php' method='post'> 
                <input type='hidden' id='_id' name='_id' value='".$id."'>
                <input type='hidden' id='title' name='title' value='".$row["title"]."'>"; 
                echo "<input type='submit' value='".$row["title"]."'>
            </</tr>
        </td>";
 echo "</form>";
}
}?></tbody></table>
            </h2>
            
          </div>
        </div>
        <div class="row" style=" direction : rtl;  font-family: 'NafeesRegular';font-weight: normal; font-style: normal;  
">
            <div class="col-md-auto">
                <h2 class="about-title" align="center"><?php   echo $title;?></h2>
                <p class="about-text" style="line-height: 2.8;">
                 <?php   echo $content;?>
                    </p>
                </div>
            </div>
            
    </section><!-- End About Section -->
</pre>

<?php/* include 'connect.php';

$id=$_GET['id'];
$content=$_POST['content']; //stores content of the chapter that has been clicked in the table of content
$title=$_POST['title']; 
if (isset($_POST['_id'])) {
  
    $id=$_POST['_id'];
}
//stores $id of the book which is selected
$content=$_POST['content']; //stores content of the chapter that has been clicked in the table of content
$title=$_POST['title'];     //stores title value for each chapter
$sql = "SELECT title,content FROM chapters WHERE book_id=".$id; //following snippet extracts name of chapters from the books
$result = $conn->query($sql);  
if ($result->num_rows > 0) 
{
    while($row = $result->fetch_assoc()) 
{
 //id of the book, and content each chapter storted in a hidden input of a form that is submitted to the same page when clicked on any chapter
 echo "<form action='urdu.php' method='post'> <input type='hidden' id='_id' name='_id' value='".$id."'>
        <input type='hidden' id='content' name='content' value='".$row["content"]."<input type='hidden' id='title' name='title' value='".$row["title"]."'>"; 
 echo "<h6><input type='submit' value='".$row["title"]."'></h6>";
 echo "</form>";
}
}
$sql = "SELECT name,title_image,description FROM books WHERE _id=".$id; //extracting details of book (title Image,name and description etc)
$result = $conn->query($sql);  
if ($result->num_rows > 0) 
{
    while($row = $result->fetch_assoc()) 
{
    
    echo "<img src='./assets/img/portfolio/".$row["title_image"].".jpg'></td><td>"; //displaying title image
    echo "<h1>".$row["name"]."</h1><br>".$row["description"]."</td></tr></table>";  //displaying description
    
}
}
echo "<p>".$content."</p>";  //if no chapter is clicked, $contnet is empty and if clicked, it contains content of the chapter.
*/?>
</body>
</html>


index.php
<!DOCTYPE html>
<html lang="en">
<?php include 'connect.php';?>
<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Mawdudid's Century</title>
  <meta content="" name="description">
  <meta content="" name="keywords">
  <meta name="twitter:image" content="">
  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">
  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,700i|Raleway:300,400,500,700,800" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Imperial - v5.1.0
  * Template URL: https://bootstrapmade.com/imperial-free-onepage-bootstrap-theme/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>

  <!-- ======= Hero Section ======= -->
  <section id="hero">
    <div class="hero-container">
      <div class="wow fadeIn">
        <div class="hero-logo">
          <img class="" src="assets/img/logo.png" alt="Imperial">
        </div>

        <h1><span class="typed" data-typed-items="Welcome to Mawdudi's Century, مودودی کی صدی میں خوش آمدید"></h1>
       
            <div class="actions">
          <a href="" class="btn-services"><h3 id="demo"></h3></a>
        </div>
      </div>
    </div>
  </section><!-- End Hero -->

  <!-- ======= Header ======= -->

  <!-- ======= Footer ======= -->
  <footer id="footer">
    <div class="container">
      <div class="row">
        <div class="col-md-12">
          <div class="copyright">
            &copy; Copyright <strong>Hybridoitc</strong>. All Rights Reserved
          </div>
          <div class="credits">
           
            Designed by <a href="hybridoitc.com">HYBRIDOITC</a>
          </div>
        </div>
      </div>
    </div>
  </footer><!-- End Footer -->

  <div id="preloader"></div>
  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>
  <script src="assets/vendor/typed.js/typed.min.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>
<script>
// Set the date we're counting down to
var countDownDate = new Date("May 8, 2021 15:37:25").getTime();

// Update the count down every 1 second
var x = setInterval(function() {

  // Get today's date and time
  var now = new Date().getTime();

  // Find the distance between now and the count down date
  var distance = countDownDate - now;

  // Time calculations for days, hours, minutes and seconds
  var days = Math.floor(distance / (1000 * 60 * 60 * 24));
  var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
  var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
  var seconds = Math.floor((distance % (1000 * 60)) / 1000);

  // Display the result in the element with id="demo"
  document.getElementById("demo").innerHTML = "We will be live after: <br>"+days + "days, " + hours + "hours, "
  + minutes + "mins, & " + seconds + "secs ";

  // If the count down is finished, write some text
  if (distance < 0) {
    clearInterval(x);
    document.getElementById("demo").innerHTML = "EXPIRED";
  }
}, 1000);
</script>
</body>

</html>

assets/js/main.js
/**
* Template Name: Imperial - v5.1.0
* Template URL: https://bootstrapmade.com/imperial-free-onepage-bootstrap-theme/
* Author: BootstrapMade.com
* License: https://bootstrapmade.com/license/
*/
(function() {
  "use strict";

  /**
   * Easy selector helper function
   */
  const select = (el, all = false) => {
    el = el.trim()
    if (all) {
      return [...document.querySelectorAll(el)]
    } else {
      return document.querySelector(el)
    }
  }

  /**
   * Easy event listener function
   */
  const on = (type, el, listener, all = false) => {
    let selectEl = select(el, all)
    if (selectEl) {
      if (all) {
        selectEl.forEach(e => e.addEventListener(type, listener))
      } else {
        selectEl.addEventListener(type, listener)
      }
    }
  }

  /**
   * Easy on scroll event listener 
   */
  const onscroll = (el, listener) => {
    el.addEventListener('scroll', listener)
  }

  /**
   * Navbar links active state on scroll
   */
  let navbarlinks = select('#navbar .scrollto', true)
  const navbarlinksActive = () => {
    let position = window.scrollY + 300
    navbarlinks.forEach(navbarlink => {
      if (!navbarlink.hash) return
      let section = select(navbarlink.hash)
      if (!section) return
      if (position >= section.offsetTop && position <= (section.offsetTop + section.offsetHeight)) {
        navbarlink.classList.add('active')
      } else {
        navbarlink.classList.remove('active')
      }
    })
  }
  window.addEventListener('load', navbarlinksActive)
  onscroll(document, navbarlinksActive)

  /**
   * Scrolls to an element with header offset
   */
  const scrollto = (el) => {
    let header = select('#header')
    let offset = header.offsetHeight

    let elementPos = select(el).offsetTop
    window.scrollTo({
      top: elementPos - offset,
      behavior: 'smooth'
    })
  }

  /**
   * Header fixed top on scroll
   */
  let selectHeader = select('#header')
  if (selectHeader) {
    let headerOffset = selectHeader.offsetTop
    let nextElement = selectHeader.nextElementSibling
    const headerFixed = () => {
      if ((headerOffset - window.scrollY) <= 0) {
        selectHeader.classList.add('fixed-top')
        nextElement.classList.add('scrolled-offset')
      } else {
        selectHeader.classList.remove('fixed-top')
        nextElement.classList.remove('scrolled-offset')
      }
    }
    window.addEventListener('load', headerFixed)
    onscroll(document, headerFixed)
  }

  /**
   * Back to top button
   */
  let backtotop = select('.back-to-top')
  if (backtotop) {
    const toggleBacktotop = () => {
      if (window.scrollY > 100) {
        backtotop.classList.add('active')
      } else {
        backtotop.classList.remove('active')
      }
    }
    window.addEventListener('load', toggleBacktotop)
    onscroll(document, toggleBacktotop)
  }

  /**
   * Mobile nav toggle
   */
  on('click', '.mobile-nav-toggle', function(e) {
    select('#navbar').classList.toggle('navbar-mobile')
    this.classList.toggle('bi-list')
    this.classList.toggle('bi-x')
  })

  /**
   * Mobile nav dropdowns activate
   */
  on('click', '.navbar .dropdown > a', function(e) {
    if (select('#navbar').classList.contains('navbar-mobile')) {
      e.preventDefault()
      this.nextElementSibling.classList.toggle('dropdown-active')
    }
  }, true)

  /**
   * Scrool with ofset on links with a class name .scrollto
   */
  on('click', '.scrollto', function(e) {
    if (select(this.hash)) {
      e.preventDefault()

      let navbar = select('#navbar')
      if (navbar.classList.contains('navbar-mobile')) {
        navbar.classList.remove('navbar-mobile')
        let navbarToggle = select('.mobile-nav-toggle')
        navbarToggle.classList.toggle('bi-list')
        navbarToggle.classList.toggle('bi-x')
      }
      scrollto(this.hash)
    }
  }, true)

  /**
   * Scroll with ofset on page load with hash links in the url
   */
  window.addEventListener('load', () => {
    if (window.location.hash) {
      if (select(window.location.hash)) {
        scrollto(window.location.hash)
      }
    }
  });

  /**
   * Preloader
   */
  let preloader = select('#preloader');
  if (preloader) {
    window.addEventListener('load', () => {
      setTimeout(() => {
        preloader.remove()
      }, 100);
    });
  }

  /**
   * Hero type effect
   */
  const typed = select('.typed')
  if (typed) {
    let typed_strings = typed.getAttribute('data-typed-items')
    typed_strings = typed_strings.split(',')
    new Typed('.typed', {
      strings: typed_strings,
      loop: true,
      typeSpeed: 100,
      backSpeed: 50,
      backDelay: 2000
    });
  }

  /**
   * Porfolio isotope and filter
   */
  window.addEventListener('load', () => {
    let portfolioContainer = select('.mawdudi-container');
    if (portfolioContainer) {
      let portfolioIsotope = new Isotope(portfolioContainer, {
        itemSelector: '.mawdudi-item',
        layoutMode: 'fitRows'
      });

      let portfolioFilters = select('#mawdudi-flters li', true);

      on('click', '#mawdudi-flters li', function(e) {
        e.preventDefault();
        portfolioFilters.forEach(function(el) {
          el.classList.remove('filter-active');
        });
        this.classList.add('filter-active');

        portfolioIsotope.arrange({
          filter: this.getAttribute('data-filter')
        });
      }, true);
    }

  });

  /**
   * Initiate portfolio lightbox 
   */
  const portfolioLightbox = GLightbox({
    selector: '.mawdudi-lightbox'
  });

  /**
   * Portfolio details slider
   */
  new Swiper('.mawdudi-details-slider', {
    speed: 400,
    loop: true,
    autoplay: {
      delay: 5000,
      disableOnInteraction: false
    },
    pagination: {
      el: '.swiper-pagination',
      type: 'bullets',
      clickable: true
    }
  });

})()